src/main/java/fi/otavanopisto/kuntaapi/server/discover/EntityEvictor.java

```diff
@@ -0,0 +1,11 @@
+package fi.otavanopisto.kuntaapi.server.discover;
+
+public abstract class EntityEvictor {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert the abstract class "EntityEvictor" into an interface. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1610)